tests/test262-esnext-excludelist.xml

```diff
@@ -10106,4 +10099,20 @@
   <test id="built-ins/TypedArrayConstructors/internals/Set/detached-buffer.js"><reason></reason></test>
   <test id="built-ins/TypedArrayConstructors/internals/Set/tonumber-value-detached-buffer.js"><reason></reason></test>
   <!-- END - Missing test262 support in JerryScript REPL - missing $262.detachArrayBuffer function -->
+  <test id="built-ins/BigInt/prototype/toString/length.js"><reason></reason></test>
+  <test id="built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-flags-u.js"><reason></reason></test>
+  <test id="built-ins/RegExp/CharacterClassEscapes/character-class-non-digit-class-escape-plus-quantifier-flags-u.js"><reason></reason></test>
+  <test id="built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape-flags-u.js"><reason></reason></test>
+  <test id="built-ins/RegExp/CharacterClassEscapes/character-class-non-whitespace-class-escape-plus-quantifier-flags-u.js"><reason></reason></test>
+  <test id="built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-flags-u.js"><reason></reason></test>
+  <test id="built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-plus-quantifier-flags-u.js"><reason></reason></test>
+  <test id="built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape-plus-quantifier.js"><reason></reason></test>
+  <test id="built-ins/RegExp/CharacterClassEscapes/character-class-non-word-class-escape.js"><reason></reason></test>
+  <test id="annexB/built-ins/Date/prototype/setYear/this-time-nan.js"><reason></reason></test>
+  <test id="annexB/built-ins/Date/prototype/setYear/this-time-valid.js"><reason></reason></test>
+  <test id="annexB/built-ins/Date/prototype/setYear/year-number-absolute.js"><reason></reason></test>
+  <test id="built-ins/JSON/stringify/value-bigint-tojson.js"><reason></reason></test>
+  <test id="built-ins/String/prototype/indexOf/searchstring-tostring-bigint.js"><reason></reason></test>
+  <test id="built-ins/decodeURI/S15.1.3.1_A2.5_T1.js"><reason></reason></test>
+  <test id="built-ins/decodeURIComponent/S15.1.3.2_A2.5_T1.js"><reason></reason></test>
 </excludeList>
```

Comment:
I don't think these test shoud fail. Maybe you have run tests on Ubuntu 18.04 with buggy timeout util.
( see https://github.com/jerryscript-project/jerryscript/pull/4232 for details )